Automattic/wp-calypso
# New Site Migration: Add Initial flow structure

Related to #[#93258](https://github.com/Automattic/wp-calypso/issues/93258)

## Proposed Changes

* Introduce the new site migration flow structure 

## Testing Instructions
* Access `/setup/hosted-site-migration-v2`
* Check if you can see the first flow step 'The plan you need'


## Pre-merge Checklist
